SR2021/2021-01-12-team-numbers.md

```diff
@@ -0,0 +1,16 @@
+---
+to: SR2021 institutions with inactive teams
+subject: Student Robotics Teams
+---
+
+Hi,
+
+If you're reading this email, it means you have at least 1 team from your institution who seem inactive on Discord, or who have few members.
```

Comment:
Presumably this is actually only going to institutions that have multiple teams assigned? I suggest that we probably want a slightly different email if they only have the one team and aren't active?

Reply:
Yeah this email is more about culling institutions with too many teams. Inactivity is definitely better measured after the league.

Reply:
Maybe up the the To then?